### DISCUSSION TITLE
this project is too large that make my idea react slow

### DISCUSSION THREAD

**Discussion by will-zdu** [open-telemetry/opentelemetry-java-instrumentation#9331] [Q&A] [answered]
My desktop is CPU:AMD Ryzen 7 6800H Memory:32GB
  idea heap memory size 8GB,bug when I use idea to open this project ,react too slow,have any suggestion to help me speed up my idea

**Comment by trask** [comment, marked as answer]
ya, unfortunately this project has a really large number of dependencies (because of all the libraries it instruments, and all of their transitive dependencies), which makes Intellij very slow initially with indexing (and then everytime you rebase and it re-indexes again) 😢

**Comment by trask** [comment]
See #9757
